startLocalDockerDB.sh

```diff
@@ -0,0 +1 @@
+docker run --name postgres-swiftpackageindex -e POSTGRES_DB=swiftpackageindex_dev -e POSTGRES_USER=swiftpackageindex -e POSTGRES_PASSWORD=password -p 5432:5432 -d postgres
```

Comment:
Why not a `Docker-Compose` file, as that scales a bit better if/when more services are added?

Reply:
Mainly laziness! It's a copy of a script I use for most repos to spin up a DB

Reply:
~shocked Pickachu~ 😛

Reply:
I started with Docker but quickly got tied in a knot trying to get it to run both a dev and test database at the same time, so I reverted to just running Postgres.app locally. That said, it's fine for this to go in if it helps you.

Reply:
Multiple databases is not something the Postgres container natively supports (though it can be added on with some scripts). Personally I just spin up multiple containers and use different ports for dev and test